.github/workflows/check-versions-updates-cron.yml

```
@@ -0,0 +1,21 @@
+name: Check for updates to versions.yaml file cron job
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Updatecli in the runner
+        uses: updatecli/updatecli-action@v2
+
+      - name: Run Updatecli in Apply mode
+        run: updatecli apply --config updatecli-manifest.yaml
+        env:
+          UPDATECLI_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          UPDATECLI_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

This environment variable is not used in the Updatecli manifest.
You are directly using `GITHUB_TOKEN` and `GITHUB_ACTOR`

The reason why I am usually using `UPDATECLI_GITHUB_TOKEN` and `UPDATECLI_GITHUB_ACTOR` is because it allows me to use other credential than the one created in the workflow which may lack permission on some Github Api.
Which I think is fine here, because the trigger a cron

---

good point, I've dropped this line